server/node-postgres-mydb/seed.json

```diff
@@ -0,0 +1,213 @@
+[{
```

Comment:
We tried to gitignore this one. Somehow it still ended on github :( Now you need to do a `git rm seed.json` to get it out.